pkg/signature/jws.go

```diff
@@ -24,6 +24,9 @@ func NewSignerFromFiles(keyPath, certPath string) (*jws.Signer, error) {
 	if err != nil {
 		return nil, err
 	}
+	if len(cert.Certificate) == 0 {
+		return nil, errors.New("missing signer certificate chain")
```

Comment:
```suggestion
		return nil, fmt.Errorf("%q does not contain a signer certificate chain", certPath)
```

Reply:
Ideally we should push line 22 onward in this method into notation-go, any cert parse/validate should be done in lower layers.

Reply:
Agree, but I would leave this change until https://github.com/notaryproject/notation-go/issues/36 is implemented, else we will be moving functions up and down without a clear target.